src/common/Screen.js

```diff
@@ -77,6 +80,7 @@ class Screen extends PureComponent<Props> {
             contentContainerStyle={[
               componentStyles.childrenWrapper && centerContent ? componentStyles.content : null,
             ]}
+            keyboardShouldPersistTaps={keyboardShouldPersistTaps}
```

Comment:
One more piece of information would be super helpful to add to the commit message: *why* are you making this change?

This might be just a sentence (in a paragraph in the body of the commit message) summarizing the relevant part of the upstream docs for this prop. In that case, it saves the reader from going and tracking down those docs when they just want to understand what you were trying to accomplish. :-)

If I understand you right, I think one version of this could be

> By default, when a text input inside a ScrollView is focused and the user taps outside it, the keyboard is dismissed and the tap doesn't reach any children of the ScrollView. On certain screens in the onboarding and login flow, we want to disable this behavior so that the user can tap directly on a button to submit the input. Add a prop to control this.

Reply:
In fact, your next couple of commits have a lot of this information already, which is great. :-) They remind me that a link to the docs for the prop is nice to include too. This commit would actually be an even better place for that than the next two, for a couple of reasons:
* This commit is the one that actually passes the prop to `ScrollView`, which is what makes it clear why the `ScrollView` docs are relevant.
* When looking at this commit, the code from the others isn't present yet, so the context is hard to infer even if the reader digs around a bit.